packages/zoe/test/unitTests/contracts/test-sellTickets.js

```diff
@@ -0,0 +1,573 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from 'tape-promise/tape';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import bundleSource from '@agoric/bundle-source';
+import harden from '@agoric/harden';
+import produceIssuer from '@agoric/ertp';
+import { E } from '@agoric/eventual-send';
+
+import { makeZoe } from '../../../src/zoe';
+import { makeGetInstanceHandle } from '../../../src/clientSupport';
+import { defaultAcceptanceMsg } from '../../../src/contractSupport';
+
+const mintAndSellNFTRoot = `${__dirname}/../../../src/contracts/mintAndSellNFT`;
+const sellItemsRoot = `${__dirname}/../../../src/contracts/sellItems`;
+
+test(`mint and sell tickets for multiple shows`, async t => {
+  // Setup initial conditions
+  const zoe = makeZoe({ require });
+  const inviteIssuer = E(zoe).getInviteIssuer();
+  const getInstanceHandle = makeGetInstanceHandle(inviteIssuer);
+
+  const mintAndSellNFTBundle = await bundleSource(mintAndSellNFTRoot);
+  const mintAndSellNFTInstallationHandle = await E(zoe).install(
+    mintAndSellNFTBundle.source,
+    mintAndSellNFTBundle.moduleFormat,
+  );
+
+  const sellItemsBundle = await bundleSource(sellItemsRoot);
+  const sellItemsInstallationHandle = await E(zoe).install(
+    sellItemsBundle.source,
+    sellItemsBundle.moduleFormat,
+  );
+
+  const { issuer: moolaIssuer, amountMath: moolaAmountMath } = produceIssuer(
+    'moola',
+  );
+
+  const invite = await E(zoe).makeInstance(mintAndSellNFTInstallationHandle);
+  const instanceHandle = await getInstanceHandle(invite);
+  const { publicAPI } = await E(zoe).getInstanceRecord(instanceHandle);
+
+  const ticketIssuer = await E(publicAPI).getTokenIssuer();
```

Comment:
Would you mind moving this down closer to its first use? 
And does this need to use `await`? `ticketIssuer` is only used with `E()`.